srcpkgs/youtube-dl/template

```diff
@@ -1,37 +1,25 @@
 # Template file for 'youtube-dl'
 pkgname=youtube-dl
 version=2019.07.27
-revision=1
+revision=2
 archs=noarch
 wrksrc="$pkgname"
-build_style=python-module
+build_style=python3-module
 pycompile_module="youtube_dl"
-hostmakedepends="python python3"
-depends="python"
-short_desc="CLI program to download videos from YouTube and other sites (Python2)"
+hostmakedepends="python3"
+depends="python3 ffmpeg"
+short_desc="CLI program to download videos from YouTube and other sites"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
 license="Unlicense"
 homepage="https://rg3.github.io/youtube-dl/"
 changelog="https://raw.githubusercontent.com/rg3/youtube-dl/master/ChangeLog"
 distfiles="https://yt-dl.org/downloads/${version}/${pkgname}-${version}.tar.gz"
 checksum=cdfc691ca9dcc63addb52dbbadb1b9e86325dd2d53d38ac844d504f482468bcc
-alternatives="youtube-dl:youtube-dl:/usr/bin/youtube-dl2"
+replaces="python3-youtube-dl>=0"
 
 post_install() {
 	rm -rf ${DESTDIR}/usr/etc
 	vinstall ${pkgname}.bash-completion 644 usr/share/bash-completion/completions ${pkgname}
 	vinstall ${pkgname}.zsh 644 usr/share/zsh/site-functions _${pkgname}
 	vinstall ${pkgname}.fish 644 usr/share/fish/completions ${pkgname}.fish
 }
-
-python3-youtube-dl_package() {
-	archs=noarch
-	depends="python3"
-	pycompile_module="youtube_dl"
-	alternatives="youtube-dl:youtube-dl:/usr/bin/youtube-dl3"
-	short_desc="${short_desc/2/3}"
-	pkg_install() {
-		vmove usr/bin/youtube-dl3
-		vmove usr/lib/python3*
-	}
-}
```

Comment:
This will cut off all users that have youtube-dl installed from any further updates. Instead we should make `python3-youtube-dl` an empty package with a dependency to `youtube-dl` and describe the behavior in `short_desc` of the subpkg.

Reply:
Ah, I misread the pkg, it's fine.

Reply:
Yeah, people with youtube-dl should just get a newer youtube-dl. people with python3-youtube-dl should get youtube-dl, if replaces works correctly.

Reply:
replaces works only if you have the package that replaces another one in the transaction.
So normal update wouldn't replace the package, `xbps-install youtube-dl` would work.

Reply:
*If you only have `python3-youtube-dl` installed its not going to be updated just because you add `replaces="python3-youtube-dl>0"` to `youtube-dl`, because xbps doesn't even look at `youtube-dl`.

Reply:
Ah, okay. I'll make python3-youtube-dl a meta/transition package then

Reply:
Okay, this should be better.